docker-compose.yml

```diff
@@ -412,7 +412,7 @@ services:
         PACKAGE: firezone-relay
     image: ${RELAY_IMAGE:-us-east1-docker.pkg.dev/firezone-staging/firezone/dev/relay}:${RELAY_TAG:-main}
     healthcheck:
-      test: ["CMD-SHELL", "lsof -i UDP | grep firezone-relay"]
+      test: ["CMD-SHELL", "curl -f localhost:8080/healthz"]
```

Comment:
Nit: Can we use the long-form of parameters in scripts? Takes some of the guess-work away.

Reply:
updated